grails-app/controllers/au/org/emii/portal/DownloadController.groovy

```diff
@@ -174,10 +174,14 @@ class DownloadController extends RequestProxyingController {
                 if (fieldIndex < currentRow.length) {
 
                     def rowValue = currentRow[fieldIndex].trim()
-                    rowValue = rowValue.replace(prefixToRemove, newUrlBase)
+                    urlSubstitutions.each {
+                        search, replace ->
+
+                        rowValue = rowValue.replaceAll(search, replace)
+                    }
 
                     if (rowValue && includedUrls.add(rowValue)) {
-                        outputWriter.print "$rowValue\n"
```

Comment:
Do you think I should change it back?  It first came in with https://github.com/aodn/aodn-portal/commit/f9c0ec81caa8c7e7fcf69be33198a48cb7a43c0e, which implies that it wasn't `println` previously, so I *think* this should be ok (better perhaps, since it's platform independent).